plugin/loadbalance/README.md

```diff
@@ -31,3 +52,49 @@ Load balance replies coming back from Google Public DNS:
     forward . 8.8.8.8 8.8.4.4
 }
 ~~~
+
+Use weighted round robin strategy to load balance replies defined by the **file** plugin:
+
+~~~ corefile
+example.com {
+        file ./db.example.com {
+                reload 10s
+        }
+        loadbalance weighted_round_robin ./db.example.com.weights {
+                    reload 10s
+        }
+}
+~~~
+
+where the weight file `./db.example.com.weights` contains:
+
+~~~
+www.example.com
+100.64.1.1 3
+100.64.1.2 1
+100.64.1.3 2
+~~~
+
+This assigns weight vales `3`, `1` and `2` to the IPs `100.64.1.1`, `100.64.1.2` and `100.64.1.3`, respectively. These IPs are addresses in A records for the domain name `www.example.com`. (The IPs for the A records are defined in the `./db.example.com` file using the **file** plugin in this example).
+
+In this example, the ratio between the number of answers in which `100.64.1.1`, `100.64.1.2` or `100.64.1.3` is in the top (first) A record should converge to  `3 : 1 : 2`.  (E.g. there should be twice as many answers with `100.64.1.3` in the top A record than with `100.64.1.2`).
```

Comment:
I think this description of what the example accomplishes should be stated before the example Corefile and weight file (i.e. moved to line 56 and maybe rephrased a bit).

Reply:
Done.